src/test/java/io/grpc/annotations/checkers/ExperimentalApiCheckerTest.java

```diff
@@ -0,0 +1,498 @@
+/*
+ * Copyright 2018, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.annotations.checkers;
+
+import com.google.errorprone.CompilationTestHelper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class ExperimentalApiCheckerTest {
+
+  private CompilationTestHelper compiler;
+
+  @Before
+  public void setUp() {
+    compiler = CompilationTestHelper.newInstance(ExperimentalApiChecker.class, getClass());
+
+    // add the @Internal annotation
+    compiler.addSourceLines("io/grpc/Internal.java",
+        "package io.grpc;",
+        "import java.lang.annotation.Documented;",
+        "import java.lang.annotation.ElementType;",
+        "import java.lang.annotation.Retention;",
+        "import java.lang.annotation.RetentionPolicy;",
+        "import java.lang.annotation.Target;",
+        "",
+        "@Internal",
```

Comment:
This can be deleted now that it's corrected upstream (https://github.com/grpc/grpc-java/pull/4051)

Reply:
Actually, now that `@Experimental` is not itself annotated as `@Internal`, the `Internal.java` source here can be dropped entirely (sorry for the headache around our annotation updates)

Reply:
Synced.